os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -1592,6 +1596,55 @@ static uint32_t i2s_txdatawidth(struct i2s_dev_s *dev, int bits)
 
 /***************************************************************************
 ****************************************************************************/
+static int i2s_pause(struct i2s_dev_s *dev)
+{
+	struct s5j_i2s_s *priv = (struct s5j_i2s_s *)dev;
+	irqstate_t flags;
+	DEBUGASSERT(priv);
+
+#ifdef I2S_HAVE_TX_P
+	if (priv->txpenab) {
+		flags = irqsave();
+		modifyreg32(priv->base + S5J_I2S_CON, 0, I2S_CR_TXCHPAUSE);
+		irqrestore(flags);
+	}
+#endif
+
+#ifdef I2S_HAVE_RX
+	if (priv->rxenab) {
+		flags = irqsave();
+		modifyreg32(priv->base + S5J_I2S_CON, 0, I2S_CR_RXCHPAUSE);
```

Comment:
@GalkinIvan If this dma is paused, then data wont flow, would n't that trigger I2S OVF/UND interrupt?

Reply:
If channel operation is paused, no data is transferred, no OVF/UND should happen.